data.json

```diff
@@ -260,6 +260,24 @@
     "dateAdded": "2018-10-24T11:42:09.000Z",
     "tags": ["clothing", "stickers"]
   },
+  {
+    "name": "Geeksforgeeks",
+    "difficulty": "easy",
```

Comment:
This seems to be a more community-oriented swag offering. I think we should focus on swag offerings designed explicitly for individual contributors. Closing this PR.